Doc/conf.py

```diff
@@ -132,6 +132,8 @@
     ('c:func', 'vsnprintf'),
     # Standard C types
     ('c:type', 'FILE'),
+    ('c:type', 'int8_t'),
```

Comment:
@encukou, I would like this (ditto for the rest of "standard C functions" changes) to be backported for CPython >=3.12.  That's for https://github.com/python/peps/pull/3975.

Does it make sense to you?  May I do pr?